core/domain/wipeout_service.py

```diff
@@ -72,6 +72,49 @@ def get_pending_deletion_request(user_id):
     )
 
 
+def get_pending_deletion_request(user_id):
+    """Return the pending deletion request.
+
+    Args:
+        user_id: str. The unique ID of the user.
+
+    Returns:
+        PendingDeletionRequest. The pending deletion request domain object.
+    """
+    pending_deletion_request_model = (
+        user_models.PendingDeletionRequestModel.get_by_id(user_id))
+    return _transform_pending_deletion_request(pending_deletion_request_model)
+
+
+def get_pending_deletion_request_by_email(email):
```

Comment:
May not be needed if we end up doing this by username.

Reply:
If we used username it would have to be `get_pending_deletion_request_by_username`

Reply:
Done, removed.